libs/deps-template/resources/io/github/kit_clj/kit/src/clj/web/handler.clj

```diff
@@ -32,8 +34,13 @@
 
 (defmethod ig/init-key :router/routes
   [_ {:keys [routes]}]
-  (apply conj [] routes))
+  (vec (map
```

Comment:
mapv?

Reply:
👍 good catch. Was in a hurry to make it work and did not clean up afterwards.

Will fix tomorrow

Reply:
This is called only once when `(go)` or `(reset)` is called, so no impact on runtime. 

The check for dev is done in handler.

Reply:
Oh right, and yeah `init-key` might be a better spot to do the check, that way the code downstream doesn't have to worry about it.